manim/mobject/svg/text_mobject.py

```diff
@@ -676,8 +676,45 @@ class MarkupText(SVGMobject):
     r"""Display (non-LaTeX) text rendered using `Pango <https://pango.gnome.org/>`_.
 
     Text objects behave like a :class:`.VGroup`-like iterable of all characters
-    in the given text. In particular, slicing is possible. Text can be formatted
-    using different tags:
+    in the given text. In particular, slicing is possible.
+
+    **What is PangoMarkup?**
+
+    PangoMarkup is a small markup language like html and it helps you avoid using
+    "range of characters" while colouring or styling a piece a Text. You can use
+    this language with :class:`~.MarkupText`.
+
+    A simple example of a marked-up string might be::
+
+        <span foreground="blue" size="x-large">Blue text</span> is <i>cool</i>!"
+
+    and it can be used with :class:`~.MarkupText` as
+
+    .. manim:: MarkupExample
+        :save_last_frame:
+
+        class MarkupExample(Scene):
+            def construct(self):
+                text = MarkupText('<span foreground="blue" size="x-large">Blue text</span> is <i>cool</i>!"')
+                self.add(text)
+
+    A more elaborate example would be
+
+    .. manim:: MarkupElaborateExample
+        :save_last_frame:
+
+        class MarkupElaborateExample(Scene):
+            def construct(self):
+                text = MarkupText('<span foreground="purple">ا</span><span foreground="red">َ</span>ل<span foreground="blue">ْ</span>ع<span foreground="red">َ</span>ر<span foreground="red">َ</span>ب<span foreground="red">ِ</span>ي<span foreground="green">ّ</span><span foreground="red">َ</span>ة<span foreground="blue">ُ</span>')
+                self.add(text)
+
+    PangoMarkup can also contain XML features such as numeric character
+    entities such as ``&#169;`` for © can be used too.
```

Comment:
```suggestion
    entities. ``&#169;`` for © can be used too.
```

Reply:
I feel the original one make more sense.

Reply:
That's fair, although doubled `such as` sounds a bit weird to my ear